tools/CAT/src/helpers/date.ts

```diff
@@ -0,0 +1,7 @@
+import dayjs from 'dayjs';
+import 'dayjs/locale/uk';
+
+dayjs.locale('uk');
+
+export const getDate = (dateString) =>
```

Comment:
Thats confusing me. What about adding a more clear name for this function?
Also add summary with some example here.